sdlib/d/gc/tcache.d

```diff
@@ -154,7 +154,11 @@ public:
 		state.enterBusyState();
 		scope(exit) state.exitBusyState();
 
-		clearTLSSegments();
+		auto tls = clearTLSSegments();
```

Comment:
You can `free(tls.ptr)` unconditionally and never clear the thread cache anyways, it is being destroyed.

Reply:
Wait, there's a reason we had to do it this way.

https://github.com/snazzy-d/sdc/pull/398

I think we need to clear it. But it doesn't need a specific function.